### PR TITLE
Changelog for 0.5.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Tools for MicroProfile Changelog
 
+## [0.5.0](https://github.com/redhat-developer/vscode-microprofile/milestone/5?closed=1) (July 28, 2022)
+
+### Enhancements
```
Comment (datho7561) [inline CHANGELOG.md:5]: Make sure to include any of the noteworthy enhancements or bug fixes from the new release of lsp4mp

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (datho7561) [reply CHANGELOG.md:5]: My bad, looks like you've got all of them

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks, Roland!

**Comment by rgrunber** [comment]
Let's revisit the `other` changes and re-categorize them.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+### Bug Fixes
+
+ * Language Server attempts to calculate code actions for stale diagnostics. See [eclipse/lsp4mp#272](https://github.com/eclipse/lsp4mp/issues/272).
+ * Hovering property value fails with NPE. See [eclipse/lsp4mp#265](https://github.com/eclipse/lsp4mp/issues/265).
+ * Completing property name with existing value will replace current value with default value. See [eclipse/lsp4mp#264](https://github.com/eclipse/lsp4mp/issues/264).
+ * Empty completion when completion is triggered before the assign `=`. See [eclipse/lsp4mp#255](https://github.com/eclipse/lsp4mp/issues/255).
+ * Improve validation by handling some known corner cases. [eclipse/lsp4mp#249](https://github.com/eclipse/lsp4mp/issues/249), [eclipse/lsp4mp#235](https://github.com/eclipse/lsp4mp/issues/235), [eclipse/lsp4mp#233](https://github.com/eclipse/lsp4mp/issues/233), [eclipse/lsp4mp#232](https://github.com/eclipse/lsp4mp/issues/232), [eclipse/lsp4mp#228](https://github.com/eclipse/lsp4mp/issues/228).
+
+### Other
+ * Add DCO information to `CONTRIBUTING.md`. See [#99](https://github.com/redhat-developer/vscode-microprofile/issues/99).
```
Comment (datho7561) [inline CHANGELOG.md:21]: documentation maybe

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+ * Remove ejs dependency. See [#97](https://github.com/redhat-developer/vscode-microprofile/pull/97).
```
Comment (datho7561) [inline CHANGELOG.md:22]: build maybe

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+ * Improved MicroProfile property value expression diagnostic message. See [eclipse/lsp4mp#242](https://github.com/eclipse/lsp4mp/pull/242).
```
Comment (datho7561) [inline CHANGELOG.md:23]: enhancement

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
 # Tools for MicroProfile Changelog
 
+## [0.5.0](https://github.com/redhat-developer/vscode-microprofile/milestone/5?closed=1) (July 28, 2022)
```
Comment (datho7561) [inline CHANGELOG.md:3]: Update the date to the 25th

**Pull request merged**